reana_cluster/backends/kubernetes/k8s.py

```diff
@@ -557,8 +557,9 @@ def verify_components(self):
         deployed component matches to docker image specified in REANA cluster
         specification file.
 
-        :return: `True` if verification of components was successful.
-        :rtype: bool
+        :return: Dictionary with component names for keys and booleans
```

Comment:
I'm not sure if this is the right way to write the docstring for the return value